### ISSUE TITLE
dev-libs/libdigidoc-3.10.4 installs Doxygen documentation to incorrect location

### ISSUE THREAD

**Issue by jaakristioja** [open-eid/gentoo#33]
```
 * The ebuild is installing to one or more unexpected paths:
 *
 *   /usr/share/doc/libdigidoc
 *
 * Please fix the ebuild to use correct FHS/Gentoo policy paths.
```
I suggest the Doxygen files be installed to /usr/share/doc/libdigidoc-3.10.4/html/ or similar instead.

**Comment by urmet** [comment]
Thanks for reporting. I didn't even have doxygen installed and never thought about it. Will fix soon and add USE flag for docs.

**Comment by urmet** [comment]
should be fixed by 8f3d23a
